tests/PeriodTest.php

```diff
@@ -92,7 +108,7 @@ public function overlappingDates(): array
              * A    [====]
              * B    [====]
              */
-            [Period::make('2018-01-01', '2018-02-01'), Period::make('2018-01-01', '2018-02-01')],
```

Comment:
Could you undo these styling changes?

Reply:
Rolled back last commit to undo changes.